src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+@import './variables';
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &__title {
+    color: $color-main;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover {
+    transform: scale(1.02);
+    transition-duration: 300ms;
+
+    .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: auto auto 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: $color-secondary;
```

Comment:
Ensure that the variable `$color-secondary` is defined in the imported `variables` file. If it's not defined, this will cause a compilation error.